codec/codec-jackson/src/main/java/com/alipay/sofa/rpc/codec/jackson/JacksonSerializer.java

```diff
@@ -72,6 +72,8 @@
 
     private static final String SERIALIZATIONFEATURE_PREFIX   = "sofa.rpc.codec.jackson.SerializationFeature.";
 
+    private static Map<String, String> genericServiceMap = new HashMap<>();
```

Comment:
Are there concurrent operations on this map?

Reply:
this map is used when application start, we can not use it in concurrent scenarios